src/message_parser.ts

```diff
@@ -4,6 +4,50 @@ const emoji = new EmojiConvertor();
 emoji.replace_mode = 'unified';
 emoji.allow_native = true;
 
+export type SlackFile = {
+  id: string;
+  created: number;
+  timestamp: number;
+  name: string;
+  title: string;
+  mimetype: string;
+  filetype: string;
+  pretty_type: string;
+  user: string;
+  editable: boolean;
+  size: number;
+  mode: string;
+  is_external: boolean;
+  external_type: string;
+  is_public: boolean;
+  public_url_shared: boolean;
+  display_as_bot: boolean;
+  username: string;
+  url_private: string;
+  url_private_download: string;
+  thumb_64: string;
+  thumb_80: string;
+  thumb_360: string;
+  thumb_360_w: number;
+  thumb_360_h: number;
+  thumb_480: string;
+  thumb_160: string;
+  original_w: number;
+  original_h: number;
+  thumb_tiny: string;
+  permalink: string;
+  permalink_public: string;
+  has_rich_preview: boolean;
+};
+
+// From: https://www.twilio.com/docs/sms/accepted-mime-types#supported-mime-types
+const SUPPORTED_MMS_MIME_TYPES = ['image/jpeg', 'image/gif', 'image/png'];
+
+// Documented size limit is 5MB. To give us a bit of buffer, we assume they
+// calculate MB with 1000 instead of 1024, and we limit to 90% of this
+// hard cap.
+const MMS_MAX_SIZE = 1000 * 1000 * 5 * 0.9;
+
 export default {
```

Comment:
@benweissmann might know, but should we remove the `export default` now that we have multiple exports?

Reply:
seems more consistent w/ the rest of the codebase.. will change!